p/themes/Origine/origine.css

```diff
@@ -1195,6 +1196,13 @@ a.btn {
 		width: 400px;
 	}
 
+	.form-group.form-actions {
+		margin-left: -15px;
+		margin-right: -15px;
+		padding-left: 15px;
+		padding-right: 15px;
+	}
```

Comment:
What about `.form-group.form-actions .btn:first-child {margin:0}`?

Reply:
Or something like

```css
.form-group.form-actions .btn {
	margin: 0;
}
.form-group.form-actions .btn+.btn {
	margin: 0 10px;
}
```

Reply:
this "green" code brings the grey background to the window border.

your code does it not.

I will mark you in the code where we should discuss your idea

Reply:
Not quite, I overlooked that you were doing things to more than just buttons. That being said, I would posit something like this as a possible alternative.

```
background: linear-gradient(90deg, rgba(0,0,0,0) 0%, rgb(244, 244, 244) 10%,rgb(244, 244, 244) 90%, rgba(0,0,0,0) 100%);
```

![Screenshot_2021-08-23_23-06-54](https://user-images.githubusercontent.com/202757/130519479-de8a2114-3dc7-4f1f-be66-e2eea55aaf43.png)

Reply:
I do not understand why you posted this code line for background

Reply:
Just thinking aloud. It's an alternative way to approach the issue. A potential advantage would be that you don't need different dedicated workarounds for desktop and mobile, but of course the looks are a matter of taste.

Reply:
I wanted to make the buttons flush to the inputs. That was my main goal. The margin-paddings are just to reach the goal in mobile view.

Reply:
If it's just to make the buttons flush I'd prefer what I said above with for example :first-child or .btn+.btn.

Reply:
your solution does not make it beautiful in mobile view, because the grey background has on the left no padding to the button with your solution.

Reply:
one more point: yes, it was not the goal to change the background, but the new layout I like more than the old one.

Reply:
So also do it on larger widths? ^_^

Reply:
I do not understand what you mean.

you see above the results of the CSS changes. What do you want to change?

Reply:
It would seem that logically you would want to add:

```
margin-left: -50px;
padding-left: 50px;
margin-right: -50px;
padding-right: 50px;
```

for larger screen widths.

Reply:
have you seen that these lines are only for the mobile view?

`@media (max-width: 840px) `

There are 15px between the gray background to the window border. That is why I used -15px/+15px.

Origine theme does not support "larger screen widths" only mobile (<840px) and normal (>840px)

Reply:
> have you seen that these lines are only for the mobile view?

That's my point…

Reply:
![Screenshot_2021-08-25 Display · FreshRSS](https://user-images.githubusercontent.com/202757/130781138-96270a4e-a852-4dcb-9c94-86a0beed7183.png)

Reply:
Thanks, now I understand your point.

I did not want to change/fix it, because it breaks the layout and is not an upgrade (imho).